Makefile

```diff
@@ -545,3 +552,6 @@ endif
 ## Help documentatin à la https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 help:
 	@grep -E '^[0-9a-zA-Z_-]+:.*?## .*$$' ./Makefile | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@echo
```

Comment:
nit: This is a "one-time" learning event. Could we just document this inline within the `Makefile` (and on mattermost.dev?), instead of permanently polluting `make help`'s output?

Reply:
We could, but I don't buy the "one-time" learning event, we could say exactly the same for any other Makefile task (you only needs to learn what `run` does one time, why is showing it to me every single time?)

Reply:
I'd go with Jesus here. I think it's useful to have this note.